src/main/java/duke/Parser.java

```diff
@@ -1,7 +1,8 @@
 package duke;
 
-
+//CHECKSTYLE:OFF
 import duke.exceptions.*;
```

Comment:
Here might be better to follow the Java coding standard. The usage of wildcard import is not recommended.